thunder/core/transforms.py

```diff
@@ -2290,6 +2301,19 @@ def iter_bound_symbols(bound_symbols):
             yield symbol
 
 
+def get_first_proxy(proxies) -> Proxy | None:
+    """Get the first proxy from a list of proxies.
+
+    Args:
+        proxies (List[Proxy]): List of proxies
```

Comment:
I'd prefer this style but it's up to you
```suggestion
def get_first_proxy(proxies: list[Proxy]) -> Proxy | None:
    """Get the first proxy from a list of proxies.

    Args:
        proxies: List of proxies
```